src/broker/overlay.c

```diff
@@ -777,9 +777,12 @@ static int overlay_sendmsg_child (struct overlay *ov, const flux_msg_t *msg)
             && (child = child_lookup_online (ov, uuid))) {
             flux_log (ov->h,
                       LOG_ERR,
-                      "%s (rank %d) transitioning to LOST due to %s",
```

Comment:
Commit message: till :arrow_right: still?

Reply:
Fixed that and another typo.  Doing some manual testing now.